Dockerfile

```diff
@@ -7,20 +7,9 @@ RUN yum install -y  \
       bash \
       go
 
-RUN mkdir /aws-sigv4-proxy
-WORKDIR /aws-sigv4-proxy
-COPY go.mod .
-COPY go.sum .
-
-RUN go env -w GOPROXY=direct
-RUN go mod download
-COPY . .
-
-RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/aws-sigv4-proxy
-
-FROM alpine:3.7
+FROM alpine:3
 COPY --from=build /etc/ssl/certs/ca-bundle.crt /etc/ssl/certs/
-COPY --from=build /go/bin/aws-sigv4-proxy /go/bin/aws-sigv4-proxy
+COPY bin/aws-sigv4-proxy /aws-sigv4-proxy
```

Comment:
As you are using `actions/download-artifact@v4`, will might need this, no? `--chmod=755`

Reply:
Thanks I forgot about that!